package.json

```diff
@@ -24,7 +24,7 @@
   "devDependencies": {
     "broccoli-asset-rev": "^2.0.2",
     "ember-cli": "1.13.1",
-    "ember-cli-app-version": "0.4.0",
+    "ember-cli-app-version": "0.3.5",
```

Comment:
`ember-cli-app-version: 0.4.0` makes use of [instanceInitializers](http://emberjs.com/blog/2015/05/13/ember-1-12-released.html#toc_instance-initializers) which were introduced in `1.12.0`.